updates/version.yaml

```diff
@@ -104,4 +104,8 @@
     - Rebrand to Winter.Translate
     - Fix location for dropdown-to in css file
     - v2.0.0/rename_tables.php
-
+"2.1.0":
+    - |
```

Comment:
What's with the bar character? Better to just have it be one line for version descriptions.

Reply:
It's more obvious when the migration runs with the output on two lines.

The second line has the "See https://github.com/wintercms/wn-translate-plugin/pull/2"